content/features/02-graph-zoomed-view.adoc

```diff
@@ -0,0 +1,10 @@
+---
+title: "Graph: Detailed View"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+image: features/graph-detailed.png
+subTitle: "Zoom in to see what is happening with a single component"
+weight: 02
+---
+
+Want to just focus on a simple component and get rid of everything else in the graph? Double clicking on a workload, app or service will provide you with a detailed view focusing just on that component and what it is directly communicating with.
```

Comment:
"get rid of" might imply on deletion, which is not the case. I would replace it with "hide" or "display only X"

Reply:
Ok